core/templates/dev/head/forms/formBuilder.js

```diff
@@ -665,34 +668,12 @@ oppia.factory('rteHelperService', [
       },
       getRichTextComponents: function() {
         return angular.copy(_RICH_TEXT_COMPONENTS);
-      }
-    };
-  }
-]);
-
-// Add RTE extensions to textAngular toolbar options.
-oppia.config(['$provide', function($provide) {
-  $provide.decorator('taOptions', [
-    '$delegate', '$modal', '$timeout', 'focusService', 'taRegisterTool',
-    'rteHelperService', 'alertsService', 'explorationContextService',
-    'PAGE_CONTEXT',
-    function(
-        taOptions, $modal, $timeout, focusService, taRegisterTool,
-        rteHelperService, alertsService, explorationContextService,
-        PAGE_CONTEXT) {
-      taOptions.disableSanitizer = true;
-      taOptions.forceTextAngularSanitize = false;
-      taOptions.classes.textEditor = 'form-control oppia-rte-content';
-      taOptions.setup.textEditorSetup = function($element) {
-        $timeout(function() {
-          $element.trigger('focus');
-        });
-      };
-
-      // The refocusFn arg is a function that restores focus to the text editor
-      // after exiting the modal, and moves the cursor back to where it was
-      // before the modal was opened.
-      var _openCustomizationModal = function(
+      },
+      isInlineComponent: function(richTextComponent) {
+        var inlineComponents = ['link', 'math'];
```

Comment:
Declare at top, and add to it in the Object.keys(RTE_COMPONENT_SPECS).forEach() call? Seems like this is something that ought to be declared in the component config...